packages/forge-trpc-adapter/src/lib/resolve-procedure-call.ts

```diff
@@ -47,8 +47,8 @@ const decodeProcedureCallOptions = (
         code: 'BAD_REQUEST',
         message: `Unexpected RPC payload: ${PathReporter.failure(
           validationError
-        ).join(', ')}`,
-      }),
+        ).join(', ')}`
+      })
```

Comment:
@jwoyo, could you check if your IDE is picking up the Prettier configuration? Almost all the changes in this file are minor formatting changes. Not super critical, but would be good if we can avoid them.

Reply:
good spot. I wonder why I haven't spotted that in the first place. I applied prettier.